### ISSUE TITLE
autocomplete only specific region

### ISSUE THREAD

**Issue by Ben-Avrahami** [wobsoriano/vue-use-places-autocomplete#9]
hey
when im sending options
`
const { suggestions } = usePlacesAutocomplete(location, {
    apiOptions: {
        version: 'weekly',
    },
    autocompletionRequest: {
        region: 'US',
    },
    debounce: 500,
    minLengthAutocomplete: 3
});
`

i get TS error Object literal may only specify known properties, and 'region' does not exist in type 'AutocompletionRequest'.ts-plugin(2353) in 
autocompletionRequest: {
        region: 'US',
    },

and also i get returned values outside of the us.
is it a bug? or am i doing it incorrectly? 
thanks.

**Comment by helo22** [comment]
Hi,

Similarly I am trying to send locationbias as 'ipbias' this way:
```
const { suggestions } = usePlacesAutocomplete(query, {
    debounce: 500,
    minLengthAutocomplete: 3,
    autocompletionRequest: {
        locationbias: 'ipbias'
    }
});
```
and it is not working. 

Could these options be added to the library or am I also doing it incorrectly?

Thanks in advance 🙏

**Comment by frasza** [comment]
That would be very welcome indeed!

**Comment by soylomass** [comment]
Would need this too.

**Comment by wobsoriano** [comment]
So sorry for the late reply. I think if you want to restrict the results on a specific country, you'd use [`componentRestrictions`](https://developers.google.com/maps/documentation/javascript/reference/places-autocomplete-service#AutocompletionRequest.componentRestrictions) instead:

```ts
usePlacesAutocomplete(query, {
  autocompletionRequest: {
    componentRestrictions: {
      country: 'US'
    }
  }
})
```

**Comment by soylomass** [comment]
@wobsoriano Thanks! That worked. Is it possible to use locationBias?

**Comment by wobsoriano** [comment]
The types for `locationBias` are added 4 hours ago in [`@types/google.maps` repo](https://github.com/googlemaps/js-types/commit/9a7dc2a3eb0baeb1e2d299a199fa96bb343866f0). I'll wait for next release before I release an updated version.

I believe you can still use `locationBias` though? or is it not working at all?

For your example @helo22, again sorry for replying late lol, it should be `IP_BIAS` based on [here](https://developers.google.com/maps/documentation/javascript/reference/places-service#LocationBias).

```ts
usePlacesAutocomplete(a, {
  autocompletionRequest: {
    locationBias: 'IP_BIAS'
  }
})
```

**Comment by simondaigre** [comment]
Hello, I'm facing the same issue. I tried your solution @wobsoriano, but it doesn't work.

**Comment by wobsoriano** [comment]
@simondaigre which issue? `locationBias`?

**Comment by simondaigre** [comment]
@wobsoriano Indeed, and I have a Typescript warning because locationBias is not in the AutocompletionRequest interface you provide.

**Comment by soylomass** [comment]
> The types for `locationBias` are added 4 hours ago in [`@types/google.maps` repo](https://github.com/googlemaps/js-types/commit/9a7dc2a3eb0baeb1e2d299a199fa96bb343866f0). I'll wait for next release before I release an updated version.
> 
> I believe you can still use `locationBias` though? or is it not working at all?
> 
> For your example @helo22, again sorry for replying late lol, it should be `IP_BIAS` based on [here](https://developers.google.com/maps/documentation/javascript/reference/places-service#LocationBias).
> 
> ```ts
> usePlacesAutocomplete(a, {
>   autocompletionRequest: {
>     locationBias: 'IP_BIAS'
>   }
> })
> ```

Tried a few minutes ago and it wasn't working. Calling it from an Argentine IP it was returning US addresses first.

**Comment by wobsoriano** [comment]
Published a new version with `locationBias` types and updated some internal dependencies - https://github.com/wobsoriano/vue-use-places-autocomplete/releases/tag/v0.2.3

Let me know if somehow it fixed the `locationBias` as well. If not, we can do 2 things:

1. Wait for a new `@googlemaps/js-api-loader` version
2. Load the library using script tags `<script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places"></script>` to get updated version

**Comment by simondaigre** [comment]
Same issue with 0.2.3.

**Comment by wobsoriano** [comment]
Can you try this and let me know?

```bash
npm install https://pkg.pr.new/vue-use-places-autocomplete@10
```

**Comment by simondaigre** [comment]
Still same issue. I checked network calls and I can't see any IP_BIAS param. 
I'll check with official Places autocomplete plugin how it should be called.

**Comment by helo22** [comment]
> Published a new version with `locationBias` types and updated some internal dependencies - https://github.com/wobsoriano/vue-use-places-autocomplete/releases/tag/v0.2.3
> 
> Let me know if somehow it fixed the `locationBias` as well. If not, we can do 2 things:
> 
> 1. Wait for a new `@googlemaps/js-api-loader` version
> 2. Load the library using script tags `<script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places"></script>` to get updated version

0.2.3 fixed the TS error regarding types, so thank you for that! 

However, I am still getting suggestions from Vietnam or Taiwan when calling it from Canada IP and I don't see any IP_BIAS param in the network calls either.

I tried this `npm install https://pkg.pr.new/vue-use-places-autocomplete@10` and also loaded the library using script tags `<script src="https://maps.googleapis.com/maps/api/js?key=YOUR_API_KEY&libraries=places"></script>` to get updated version.

Any clues would be appreciated!

**Comment by helo22** [comment]
Hi @wobsoriano, any updates on the IP_BIAS not working? Thanks!

**Comment by mtzrmzia** [comment]
any update on this?
